custom_model_runner/datarobot_drum/resource/pipelines/uwsgi_pipeline.json.j2

```diff
@@ -0,0 +1,20 @@
+  {
+      "name": "Python Default Predictor",
+      "engineType": "RestModelServing",
+      "pipe": [
+          {
+              "name": "DF Source",
+              "id": 1,
+              "type": "uwsgi_serving",
+              "parents": [],
+              "arguments": {
+                  "positiveClassLabel": {{ positiveClassLabel }},
+                  "negativeClassLabel": {{ negativeClassLabel }},
+                  "__custom_model_path__": "{{ customModelPath }}",
+                  "run_language": "{{ run_language }}",
+                  "port": {{ port }},
+                  "uwsgi_max_workers": 1
```

Comment:
I feel like in general amount of workers will become something that user is aware. Since they will be exposed to how much ram they use / are allowed to use, amount of workers will directly affect that.

Reply:
I set this to 1 only to run tests - one test case is failing and i what to check if this is the reason.

Cheaper system strategy itself is defined here: https://github.com/datarobot/mlpiper/blob/master/mlpiper-py/mlpiper/components/restful/uwsgi_cheaper_subsystem.py

For now I'm going to make it configurable from drum in only one way: max number of workers can be provided to drum.

Share your ideas and opinion.